README.md

```diff
@@ -19,35 +27,45 @@ git clone git@github.com:ops-guru/docker-hive.git
 Create image
 
 ```
-docker build . -t amzn-hive-image
+docker build . -t opsguruhub/docker-hive
 ```
 
+You may want to do some cleanup first:
+docker container stop local-hive; docker rm local-hive; docker rmi opsguruhub/docker-hive
+
 ## DockerHub
 
 Image available on DockerHub
 
 ```
-docker run -d opsguruhub/docker-hive:0.0.1
+docker pull opsguruhub/docker-hive:0.0.2
```

Comment:
if you want to pull the latest version, just omit `:$tag`

Reply:
Done